### PR TITLE
fixed missing add-apt-repository command when ubuntu is the server one

### PR DESCRIPTION
**Pull request by julienlavergne**

### CODE REVIEW

**Comment by babilen** [comment]
You might be delighted to learn that you can simply force push into the branch on which the PR is based and that you don't have to open/close new ones for simple changes.

**Comment by julienlavergne** [comment]
Great!

Next time I could test it.

-- 
Julien Lavergne
Sent with Airmail

On April 23, 2015 at 10:26:44 AM, Wolodja Wentland (notifications@github.com) wrote:

You might be delighted to learn that you can simply force push into the branch on which the PR is based and that you don't have to open/close new ones for simple changes.

File: dev/setup-salt.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
-sudo apt-add-repository ppa:saltstack/salt -y
+sudo apt-get update -y
```
Comment (babilen) [inline dev/setup-salt.sh]: This is a duplicate (repeated later on)

**Comment by iggy** [comment]
Thanks.

**Pull request merged**